cylc/flow/config.py

```diff
@@ -890,7 +884,9 @@ def _check_implicit_tasks(self) -> None:
             )
         # Allow implicit tasks in back-compat mode unless rose-suite.conf
         # present (to maintain compat with Rose 2019)
-        elif not Path(self.run_dir, 'rose-suite.conf').is_file():
+        elif (
```

Comment:
What was wrong with this before?

Reply:
It was always pointing at the run dir, the point of this PR is point this variable at the workflow configuration dir, which is sometimes the run dir, but could also be the src dir.

Reply:
Oh heck, is this fixing a bug?!

Reply:
On this branch, `self.run_dir` is only defined by the scheduler. (i.e., not if we're parsing the config file validation etc.). So this change just makes the `rose-suite.conf` check continue to work in the source directory as well as the run directory.

Reply:
Fair play, I actually didn't realize `/` coerces its args like that.  I'll make that change.